typify-impl/src/enums.rs

```diff
@@ -1453,6 +1453,71 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_enum_default_generation_simple_variant() {
```

Comment:
I recently added a test harness that converts json schema files into .rs files and compiles them. Could you just add this as a file in the typify/schemas/ directory? I think it will be equivalent to what you have **and** will compile the resulting output.

Reply:
Can you please also add the test case from #125 ? I think it should be fixed by this now as well.

Reply:
Nevermind: I fixed that case differently

Reply:
but please DO put THIS new test into a file in typify/tests/schemas